scripts/config-traffic-control.sh

```diff
@@ -0,0 +1,89 @@
+#!/usr/bin/env bash
+# Copyright (c) Mysten Labs, Inc.
+# Modifications Copyright (c) 2024 IOTA Stiftung
+# SPDX-License-Identifier: Apache-2.0
+
+usage() {
```

Comment:
should that script be in the `scripts` folder, or in the `setups/` folder?

Reply:
Moved in 24bf0619865185cfbeb2908a0b2accdfc68ae0ae